### ISSUE TITLE
Price check doesn't give results

### ISSUE THREAD

**Issue by Ashandaar** [SnosMe/awakened-poe-trade#1201]
When i press CTRL -D to price check my items, here's the answer  

Trade site request failed
Error: Unexpected token '<', "<!DOCTYPE "... is not valid JSON

Client is updated to the last version, was working great on yesterday, stopped working 2hrs ago

**Comment by DDavion** [comment]
Having same issue

**Comment by null77** [comment]
Yep seems to be a new thing, not sure if it's a transient error or poe changed something.

**Comment by null77** [comment]
OK, this is something to do with cloudflare. I clicked "browser", had to verify their captcha, then I was able to access prices again via awakened poe trade's ui.

**Comment by halexh** [comment]
> OK, this is something to do with cloudflare. I clicked "browser", had to verify their captcha, then I was able to access prices again via awakened poe trade's ui.

Just to add to this, try and price check something, it will fail. In the failed results click "Browser" and then pass the captcha. I was thinking "Browser" was in settings somewhere, wasnt able to initially find it.

**Comment by Ashandaar** [comment]
I came back to the game and tried, now it works properly. Troll magic :D!

**Comment by elmo667** [comment]
Yep, newly introduced Clownflare issue :/
